render.yaml

```diff
@@ -26,4 +26,5 @@ services:
         value: enabled
       - key: ROBOTS_NO_INDEX # Imported from Heroku app
         value: true
-
+      - key: TEST_ENV_VAR
```

Comment:
I tried this, but couldn't get it to work.

Reply:
Hi @lachlanhardy thanks for trying this again. I've been struggling to get Render to reliably do its thing and no luck. Mainly because I'm struggling to cognitively follow a complicated process to replicate it.

I'm thinking to just do something really simple, like `IF hostname is buildkite.com THEN use production scripts ELSE dev scripts`. Not the most elegant solution but at least it serves the purpose of not contaminating analytics data.